client/src/components/HomePage/HomepageHeader.jsx

```diff
@@ -13,10 +32,21 @@ const Header = () => {
             <input className="search-input" placeholder="Search" />
           </div>
           <div className="header-buttons">
-            <button className="header-profile">DM</button>
+            <button
+              className="header-profile"
+              onClick={(event) => handleProfileClick(event)}
+            >
+              DM
```

Comment:
Is this your initials? Lets not forget to change this to the actual user's initials later.